codacy/values.yaml

```diff
@@ -141,8 +141,8 @@ global:
 
   rabbitmq:
     create: true
-    rabbitmqUsername: rabbitmq
-    rabbitmqPassword: rabbitmq
+    rabbitmqUsername: rabbitmq-codacy
+    rabbitmqPassword: rabbitmq-codacy
```

Comment:
This new rabbitmq chart does not let us use the default credentials (rabbitmq), hence we must change them